apps/prairielearn/src/components/QuestionContainer.html.ts

```diff
@@ -305,21 +293,27 @@ export function QuestionFooter({
   if (resLocals.question.type === 'Freeform') {
     return html`
       <div class="card-footer" id="question-panel-footer">
-        ${QuestionFooterContent({ resLocals, questionContext })}
+        <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+        <div id="question-panel-footer-content">
+          ${QuestionFooterContent({ resLocals, questionContext })}
+        </div>
       </div>
     `;
   } else {
     return html`
       <div class="card-footer" id="question-panel-footer">
         <form class="question-form" name="question-form" method="POST">
-          ${QuestionFooterContent({ resLocals, questionContext })}
+          <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+          <div id="question-panel-footer-content">
+            ${QuestionFooterContent({ resLocals, questionContext })}
+          </div>
         </form>
       </div>
     `;
   }
 }
 
-function QuestionFooterContent({
+export function QuestionFooterContent({
```

Comment:
I haven't checked, but this might mean `QuestionFooter` no longer needs to be exported?

Reply:
That's probably correct, I'll handle that in cleanup later on.

Reply:
Export removed in 7b2d5ff.